container-engine-lib/lib/backend_impls/docker/docker_manager/docker_manager.go

```diff
@@ -1344,11 +1344,14 @@ func (manager *DockerManager) GetAvailableCPUAndMemory(ctx context.Context) (com
 //	INSTANCE HELPER FUNCTIONS
 //
 // =================================================================================================================
-func (manager *DockerManager) isImageAvailableLocally(ctx context.Context, imageName string) (bool, error) {
+func (manager *DockerManager) isImageAvailableLocally(imageName string) (bool, error) {
+	// Own context for checking if the image is locally available because we do not want to cancel this works in case the main context in the request is cancelled
```

Comment:
wow this i interesting case, is there an issue with context on this issue?

Reply:
yes, it's related to ticket #1768 (it fails in Ubuntu Firefox) I'm exploring if this change finally fixes the issue, because if the context is canceled during this call all the create enclave process is finished during the create enclave